spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/SparkCatalog.java

```diff
@@ -222,11 +222,11 @@ public SparkTable alterTable(Identifier ident, TableChange... changes) throws No
     try {
       Table table = load(ident);
       commitChanges(table, setLocation, setSnapshotId, pickSnapshotId, propertyChanges, schemaChanges);
+      table.refresh();
```

Comment:
This doesn't seem to be necessary. The test still pass if I remove this. Should I remove it if it saves a network round trip to talk to the catalog?

Reply:
This is unnecessary. The commit will update the table.

Reply:
I should have been more clear. Can you please remove this?